cmd/ipfs/daemon.go

```diff
@@ -457,6 +469,99 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 	return errs
 }
 
+const mfsRepinInterval = time.Minute
+
+func pinMFSOnChange(cctx *oldcmds.Context, node *core.IpfsNode) (<-chan error, error) {
+	errCh := make(chan error)
+	go func() {
+		var lastRootCid cid.Cid
+		for {
+			// poll periodically
+			tmo := time.NewTimer(mfsRepinInterval)
```

Comment:
minor: Instead of creating a new timer each iteration,
```go
var tmo *time.Timer
for {
    if tmo == nil {
        tmo = time.NewTimer(mfsRepinInterval)
    } else {
        tmo.Reset(mfsRepinInterval)
    }
    ...
```